CONTRIBUTING.md

```diff
@@ -23,7 +24,13 @@ Then, markdown file name will be ``title-of-the-blog.md``
 * The main blog image should be named as ``title-of-the-blog.png`` and add this file to ``website/website/public/images/blog``
 * If you are a new author, add your photograph with file name as `author-name.png` to `website/website/public/images/blog/authors`. Make sure the name provided in the author section of the metadata of the blog matches with this filename
 
+*Note:*
 
+* **Italic text:** To bold text, add two asterisks or underscores before and after a word or phrase. For example, `*italic text*`, `_italic text_`. Both of these texts will be rendered as *italic text*.
+* **Bold text:** To italicize text, add one asterisk or underscore before and after a word or phrase. For example, `**bold text**`, `__bold text__`. Both of these texts will be rendered as **bold text**
```

Comment:
To italicize text => To make text bold

Please revisit these two notes regarding bold and italic